tests/ipfwd/test_nhop_group.py

```diff
@@ -325,6 +325,17 @@ def build_pkt(dest_mac, ip_addr, ttl, flow_count):
     return pkt, exp_packet
 
 
+def validate_asic_route(duthost, route, exist=True):
+    logger.info(f"Checking ip route: {route}")
+    asic_info = duthost.shell(f'redis-cli -n 1 keys "ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY*" | grep {route}',
+                              module_ignore_errors=True)["stdout"]
```

Comment:
Why not put the route into the key of ASIC_DB, like this 
```redis-cli -n 1 keys "ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY:{route}```

Reply:
Just keep the original class IPRoutes unchanged. If change the way of add route, then all the methods in IPRoutes should be changed as well. It would take more efforts, so I choose to validate the static route from asic db.

Reply:
Maybe it's better to add the key to the redis command? grep is not reliable sometimes because of some unexpected output.
```
redis-cli -n 1 keys "ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY|{route}
```

Reply:
> Maybe it's better to add the key to the redis command? grep is not reliable sometimes because of some unexpected output.
> 
> ```
> redis-cli -n 1 keys "ASIC_STATE:SAI_OBJECT_TYPE_ROUTE_ENTRY|{route}
> ```

@bingwang-ms , updated as your comments.